src/drawer.js

```diff
@@ -611,14 +611,16 @@ $.Drawer.prototype = {
      */
     setImageSmoothingEnabled: function(imageSmoothingEnabled){
         if ( this.useCanvas ) {
-            this.context.mozImageSmoothingEnabled = imageSmoothingEnabled;
-            this.context.webkitImageSmoothingEnabled = imageSmoothingEnabled;
-            this.context.msImageSmoothingEnabled = imageSmoothingEnabled;
-            this.context.imageSmoothingEnabled = imageSmoothingEnabled;
+            var context = this.context;
+            context.save();
```

Comment:
Actually we definitely don't want the save/restore... it would wipe out the `imageSmoothingEnabled` change we just made. (Sorry I didn't think that through earlier).

Reply:
Ok, I removed that. But I still wonder why drawDebugInfo and others use that then. It also seems to work with the save/restore routine.